android/impl.nix

```diff
@@ -76,7 +76,7 @@ in {
         });
         javaSrc = nixpkgs.buildEnv {
           name = applicationId + "-java";
-          paths = [
+          paths = javaSources ++ [
             (ghcAndroidAarch64.android-activity.src + "/java") #TODO: Use output, not src
             (ghcAndroidAarch64.reflex-dom.src + "/java")
```

Comment:
Might we make these two the default for `javaSources`? Or maybe it's too soon to start layering `buildGradleApp` below the reflex stuff.

Reply:
As this PR is right now you can add `javaSources = [ ./java ];` to the `android` section of an Obelisk app and it just works. I'm all for properly layering `buildGradleApp` internally but the interface we expose to the project system should probably work like this for now (with `android-activity` and `reflex-dom` built-in).